apps/mobile/src/components/widgets/accounts/components/cards/account-card.layout.tsx

```diff
@@ -16,9 +20,16 @@ export function AccountCardLayout({
   width = 200,
   testID,
 }: AccountCardLayoutProps) {
+  const { pressed, onPressIn, onPressOut } = usePressedState();
+  const animatedStyle = useAnimatedStyle(() => ({
+    transform: [{ scale: withSpring(pressed ? 0.95 : 1) }],
+  }));
```

Comment:
Should simple transitions like this be put into a shared folder and reused? Can imagine elsewhere another dev does 0.85 etc instead and we have a mash of slight variants.

Reply:
100%. I should've elaborated: this is what I meant by consistent animation "framework". This should stem from design & prototyping tools of choice as well, so we can define standardizes spring properties to make the animations feel the same. I'll re-visit this along with the remaining pressed transitions.

Reply:
I would put this `animatedStyle` and `usePressedState` with the pressable component in the UI library. 

I guess we will want to apply this same animation to most instances of `Pressable`, at least as a default

Reply:
Agreed. Let me do this after I'm done with the remaining pressed transitions. Might need to do some thinking in terms of naming/structure of those as well. 
The initial idea was to add an ability for defining those more declaratively, through serializable config presets: `const animationX = { type: 'spring', damping: 5, stiffness: 20, ...etc  }` passed right into the pressable, but there are too many tooling obstacles in the way of doing this quickly, so settling with RNR API for the time being.
Will move the `usePressableState` into `ui` within this PR.

Reply:
Sounds great 👍